mne/viz/backends/_pyvista.py

```diff
@@ -25,11 +25,13 @@
     warnings.filterwarnings("ignore", category=DeprecationWarning)
     import pyvista
     from pyvista import (Plotter, BackgroundPlotter, PolyData,
-                         Line, close_all, UnstructuredGrid)
+                         Line, close_all, UnstructuredGrid,
+                         rcParams)
     from pyvista.utilities import try_callback
 
 
 _FIGURES = dict()
+rcParams["depth_peeling"]["enabled"] = False
```

Comment:
Global changes are always a bit dangerous. Can we use a context manager to set this and only do it when we instantiate the plotter that will cause the check?

Reply:
I think a context manager is maybe too much here. I will isolate the changes to `rcParams` in the `_Renderer` (where the plotter is instantiated)